### ISSUE TITLE
What to do with unconfirmed bug reports

### ISSUE THREAD

**Issue by un1versal** [kvirc/KVIrc#2052]
There are at this precise time 66 bugs reports open.

37 of those are **unconfirmed bug reports** and most are older than sin

Over the last few months Ive been working towards confirming as many as possible and managing triaging, **but without help or feedback from users and people involved in project** many of them require systems or setups I just dont have more interest in spending time on a job, lets face to help project by releasing peoples times to get so that they spend that time on important parts of developing or providing feedback on PR's etc, (thats hasnt been going well at all either)

**tl;dr** In sum the project need to move forward.

**So, my proposal is, is close everything that is not confirmed older than 5 years**

**EXCEPT**
- We keep reports that had duplicate reports merged into them reports (since they have multiple reports and likely more info) There were 89 closed duplicate reports which were merged into other things.

**We close everything unconfirmed with extreme prejudice especial NO one replied to it EVER.**

So say YEY OR NEY and Ill take care of the rest.

**Comment by wodim** [comment]
Did you label them or something? Not a mock, I want to check out the list

**Comment by Dessa** [comment]
i guess its this for starters: https://github.com/kvirc/KVIrc/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3A%22Severity%20%E2%96%A0%20Bug%22%20-label%3A%22Status%20%E2%96%A0%20Confirmed%22 though a seperate milestone has been created as well, by the looks of it not finished yet https://github.com/kvirc/KVIrc/milestone/7

**Comment by un1versal** [comment]
Youre right, I just made that milestone as I read the feedback... I went around and marked a few, but feel free to **label them confirmed** or **add or remove issues from that milestone** but **keep in mind the exceptions**.

More feedback also welcome.

Ill need a once over that list with more time but its a rought list to start

[click here Unconfirmed for for removal milestone](https://github.com/kvirc/KVIrc/milestone/7)

**Comment by un1versal** [comment]
anyone?

**Comment by wodim** [comment]
Leave them there? For example #850 is not fixed. Why close it?

**Comment by un1versal** [comment]
Because no one confirmed it and because its 2010 bug, and afaiks its not an issue either.

**Comment by wodim** [comment]
Of course it is confirmed: 

https://github.com/kvirc/KVIrc/issues/850#issuecomment-124933256

And it is a bug, since it makes our http client non-compliant

**Comment by un1versal** [comment]
I dont see anyone aside from the person reporting the issue answering to that report, self confirmation isn't confirmation of anything.

That bug imo doesnt exist currently either

And hers an aside, thats 6 year old report, if no one actually even bother to confirm it, even if that was a bug which is not (I cant duplicate it) when is it acceptable to close such issues? Never right?

**Comment by wodim** [comment]
Well, ambossarm is sort of a "trusted user". In any case the bug is alive and kicking, now transfer with redirects are not marked as failed but they are not followed anyway so we are at the same spot.

**Comment by un1versal** [comment]
I rather have bugs actually confirmed (not just by trusted users) at very least, its not much of a demanding criteria for any bug validity, now is it?

Whers this users nowdays? can we even ask him to retest?

**Comment by wodim** [comment]
So confirm them yourself. I confirmed that one to you.

![303174095_c6fa3ab2cd_o](https://cloud.githubusercontent.com/assets/1001553/16834881/2163918e-49b6-11e6-8636-713446f66bcd.png)

**Comment by un1versal** [comment]
How do you think they get confirmed otherwise? Magic?

I been managing the issue tracker for months pretty much by myself, since you cant even be bothered to label anything except to amuse yourself.

**Comment by wodim** [comment]
Well, and what can I say? For example, #1324 is not confirmed by anybody. Somebody should confirm whether it still happens or not. Until that happens (until we get a YES or a NO), the bug remains open. It's that simple. What's the trouble with having bugs open?

**Comment by un1versal** [comment]
>  For example, #1324 is not confirmed by anybody. Somebody should confirm whether it still happens or not. Until that happens (until we get a YES or a NO), the bug remains open

I made special provisions for what to close exactly in my proposal on main post of which https://github.com/kvirc/KVIrc/issues/1324 fall under the exception.

> So, my proposal is, is close everything that is not confirmed older than 5 years
> 
> EXCEPT
> 
>  We keep reports that had duplicate reports merged into them reports (since they have multiple reports and likely more info) There were 89 closed duplicate reports which were merged into other things.
> 
> #### **We close everything unconfirmed with extreme prejudice especial NO one replied to it EVER.**
> 
> What's the trouble with having bugs open?

Technically there is no trouble but after 6 years or more in some cases when bugs are reported against now unsupported versions, literally no project I ever head of keeps tickets open that long unconfirmed, eventually every project has a purge of these types of reports.

**Not only that reports from SVN you have no way to get feedback from reporters anymore. Useless if you ask me.**

**Comment by un1versal** [comment]
closing this, sod it for back seat drivers.
